engine/src/flutter/lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -171,6 +171,9 @@ extension type DomConsole._(JSObject _) implements JSObject {
   void debug(Object? arg) => _debug(arg.toString());
 }
 
+@JS('parseFloat')
+external JSNumber parseFloatImpl(JSString value);
```

Comment:
Looks like we never need the JS types, maybe `external double parseFloatImpl(String value);` to simplify instead?

Reply:
sweet!